extensions/teamgantt/package.json

```
@@ -0,0 +1,82 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "teamgantt",
+  "title": "TeamGantt",
+  "description": "Finding and creating tasks in TeamGantt",
+  "icon": "command-icon.png",
+  "author": "knownasilya",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search",
+      "title": "Search Tasks",
+      "subtitle": "Find tasks by a handful of fields like title and note",
```

```suggestion
      "subtitle": "TeamGantt",
```
And maybe some of the content from subtitle to description